src/TemplatePassthrough.js

```diff
@@ -3,35 +3,33 @@ const TemplatePath = require("./TemplatePath");
 const debug = require("debug")("Eleventy:TemplatePassthrough");
 
 class TemplatePassthrough {
-  constructor(inputPath, outputDir, inputDir) {
-    this.path = inputPath;
+  constructor(path, outputDir, inputDir) {
+    this.inputPath = path.inputPath;
+    this.outputPath = path.outputPath;
     this.outputDir = outputDir;
     this.inputDir = inputDir;
     this.isDryRun = false;
   }
 
   getOutputPath() {
-    return TemplatePath.join(
-      this.outputDir,
-      TemplatePath.stripLeadingSubPath(this.path, this.inputDir)
-    );
+    const { inputDir, outputDir, inputPath, outputPath } = this;
+    // assuming if paths are the same an outputPath was not set and we will resolve manually?
```

Comment:
See comment in PR. I don't love method of testing if the output path was set. If it's not set we resolve an output path that includes the last directory, if it is set we don't.

That means in theory these to examples _should_ have different output paths:
```
config.addPassthroughCopy("./src/site/static"); // output path: "/static"
```
```
config.addPassthroughCopy("./src/site/static", "./src/site/static"); // output path SHOULD be: "/src/site/static" however will equal "/static"
```
```
config.addPassthroughCopy("./src/site/static", "./src/site/static/deeper"); // output path: "/src/site/static/deeper"
```
You can see that adding one more directory in this **edge case** would result in a significant difference to the output path. It's an edge case and it's understandable. However If the code can be improved it probably should. (want help\feedback)